### PR TITLE
fix(domain-progress): fr domain information update

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
MBP-114

### Requirements

* The wait period for a fr domain transfer should be updated to 8 days
* If the response for the transfer is negative, it can be changed within 22 days

### Description of the Change

The above changes have been made for the .fr domain.

### CODE REVIEW

File: client/app/domain-operation/progress/domain-operation-progress.html [hunk 1]
```diff
@@ -94,8 +94,8 @@ <h3 class="mb-0" data-ng-bind=":: 'domains_operations_progress_step_' + ctrlDoma
                         </ul>
                         <ul data-ng-if="ctrlDomainOperationProgress.currentStepIndex === 1">
                             <li data-translate="domains_operations_progress_instructions_3"></li>
-                            <li data-translate="domains_operations_progress_instructions_4"></li>
-                            <li data-translate="domains_operations_progress_instructions_5"></li>
+                            <li data-translate="{{ ctrlDomainOperationProgress.domainExtension === 'fr' ? 'domains_operations_progress_instructions_fr_4' : 'domains_operations_progress_instructions_4' }}"></li>
```
Comment (JeremyDec) [inline client/app/domain-operation/progress/domain-operation-progress.html]:
```suggestion
                            <li data-translate="{{:: ctrlDomainOperationProgress.domainExtension === 'fr' ? 'domains_operations_progress_instructions_fr_4' : 'domains_operations_progress_instructions_4' }}"></li>
```
Also, `'fr'` should be a constant

File: client/app/domain-operation/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -108,6 +108,8 @@
   "domains_operations_progress_instructions_3": "Il est conseillé de valider les deux emails reçus même si l'adresse email des deux contacts est identiques.",
   "domains_operations_progress_instructions_4": "En cas de refus ou d'absence de réponse sous 5 jours : le transfert est abandonné.",
   "domains_operations_progress_instructions_5": "Si un seul des deux emails a été validé : aprés 5 jours sans validation sur l'autre email, le transfert passe à l'étape suivante.",
+  "domains_operations_progress_instructions_fr_4": "En cas de refus ou d'absence de réponse sous 8 jours : la durée totale du changement de bureau d'enregistrement est étendue à 22 jours avant que ce dernier soit réalisé. Le bureau d'enregistrement sortant peut malgré tout revenir sur sa décision et donner son accord au transfer avant la fin des 22 jours.",
```
Comment (JeremyDec) [inline client/app/domain-operation/translations/Messages_fr_FR.json]:
```suggestion
  "domains_operations_progress_instructions_fr_4": "En cas de refus ou d'absence de réponse sous 8 jours : la durée totale du changement de bureau d'enregistrement est étendue à 22 jours avant que ce dernier soit réalisé. Le bureau d'enregistrement sortant peut malgré tout revenir sur sa décision et donner son accord au transfert avant la fin des 22 jours.",
```

File: client/app/domain-operation/progress/domain-operation-progress.html (hunk 1 above)
Comment (ganeshkumar1989) [reply client/app/domain-operation/progress/domain-operation-progress.html]: The one-time binding is done. Having 'fr' as a constant is handled in another way now :)

File: client/app/domain-operation/translations/Messages_fr_FR.json (hunk 2 above)
Comment (ganeshkumar1989) [reply client/app/domain-operation/translations/Messages_fr_FR.json]: This is done. Thanks :)

File: client/app/domain-operation/progress/domain-operation-progress.controller.js [hunk 3]
```diff
@@ -78,5 +78,14 @@ angular.module('App').controller(
           this.loading = false;
         });
     }
+
+    getTranslatedContent(translationIdPrefix) {
+      const domainExtension = this.domain.split('.').pop().toLowerCase();
```
Comment (marie-j) [inline client/app/domain-operation/progress/domain-operation-progress.controller.js]:
```suggestion
      const [,domainExtension] = this.domain.split('.');
```

Comment (ganeshkumar1989) [reply client/app/domain-operation/progress/domain-operation-progress.controller.js]: Thanks! That's a good idea and its implemented

**Comment by ganeshkumar1989** [comment]
FR text has been updated as provided by the content team

**Comment by jleveugle** [comment]
Hello @ganeshkumar1989,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @ganeshkumar1989,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**